test/python/visualization/test_circuit_text_drawer.py

```diff
@@ -1407,32 +1401,6 @@ def test_text_spacing_2378(self):
         circuit.rz(11111, qr[2])
         self.assertEqual(str(circuit_drawer(circuit, output="text", initial_state=True)), expected)
 
-    @unittest.skipUnless(HAS_TWEEDLEDUM, "Tweedledum is required for these tests.")
-    def test_text_synth_no_registerless(self):
-        """Test synthesis's label when registerless=False.
-        See https://github.com/Qiskit/qiskit-terra/issues/9363"""
-        expected = "\n".join(
-            [
-                "                ",
-                "     a: |0>──■──",
-                "             │  ",
-                "     b: |0>──■──",
-                "             │  ",
-                "     c: |0>──o──",
-                "           ┌─┴─┐",
-                "return: |0>┤ X ├",
-                "           └───┘",
-            ]
-        )
-
-        @classical_function
-        def grover_oracle(a: Int1, b: Int1, c: Int1) -> Int1:
-            return a and b and not c
-
-        circuit = grover_oracle.synth(registerless=False)
-        self.assertEqual(str(circuit_drawer(circuit, output="text", initial_state=True)), expected)
```

Comment:
can we still do this tests with grover or phase oracle w/o tweedledum ?

Reply:
It doesn't make sense, since this is a test for registerless synthesis, which is currently not supported by the oracles.